### DISCUSSION TITLE
How do you go about learning Docker Enterprise?

### DISCUSSION THREAD

**Discussion by Rob-Wal** [BretFisher/ama#122] [Q&A]
Hi Bret and community,

I am an experienced sys-admin/sys-engineer who have taken both of your docker and swarm mastery courses.  I am about to run through them a second time and  I have been thinking about the next step in the journey.  How does one go about learning Docker Enterprise? Is there a free trial or demo that can be deployed?  

I know about the components but do not even know what the interface looks like. 

Also I must add that the company I work for is not close to starting our cloud or container journey, but most new jobs want those skills and engineer with experience.  What are your thoughts on this?

**Comment by BretFisher** [comment]
You can get a Docker Enterprise trial on Docker Hub. Not sure it's the 3.0 release (just happened) but I'd wait for that to put any effort into learning it. https://hub.docker.com/editions/enterprise/docker-ee-trial

Yep always keep learning, regardless of the job. See the other questions in this AMA about DevOps, Security, Jobs, etc.

**Comment by BretFisher** [comment]
I keep questions open so others can find them.

**Comment by Rob-Wal** [comment]
Ok. No problem. I was cleaning up my profile and thought I left it open.
